cmd/peek-submission/main_test.go

```diff
@@ -42,29 +39,8 @@ func TestPeekBase(test *testing.T) {
 			testCase.targetSubmission,
 		}
 
-		commonCases := cmd.CommonCMDTestCases{
-			ExpectedExitCode: testCase.expectedExitCode,
-			ExpectedStdout:   testCase.expectedStdout,
-			ExpectedStderr:   testCase.expectedStderr,
-		}
-
-		cmd.RunCommonCMDTests(test, main, args, commonCases, fmt.Sprintf("Case %d: ", i))
-	}
-}
+		cmd.RunCommonCMDTests(test, main, args, testCase.CommonCMDTestCases, fmt.Sprintf("Case %d: ", i))
 
-func TestPeekVerbose(test *testing.T) {
-	for i, testCase := range testCases {
-		args := []string{
-			testCase.targetEmail,
-			testCase.courseID,
-			testCase.assignmentID,
-			testCase.targetSubmission,
-			"--verbose",
-		}
-
-		_, _, _, err := cmd.RunCMDTest(test, main, args)
-		if err != nil {
-			test.Errorf("Case %d: CMD run returned an error: '%v'.", i, err)
-		}
+		cmd.RunVerboseCMDTests(test, main, args, fmt.Sprintf("Case %d: ", i))
```

Comment:
No, make this another test.
(I have talked about this several times, if you don't understand talk to me.)